docs/backups-storage.md

```diff
@@ -208,3 +208,65 @@ within the default memory limits.
             credentialsSecret: my-cluster-azure-secret
           ...
     ```
+
+## Remote file server
+
+Here is an example of the `deploy/cr.yaml` backup section fragment,
+which configures a private volume for filesystem-type storage:
+
+You can use `fileystem` backup storage type to mount a *remote file server* to
+a local directory as a *sidecar Volume*, and make Percona Backup for MongoDB
```

Comment:
```suggestion
a local directory as a *sidecar volume*, and make Percona Backup for MongoDB
```

Reply:
Better we leave it as is, we capitalize k8s terms to make wording more clear